lib/log-formatters.js

```diff
@@ -88,7 +88,7 @@ module.exports = function (d) {
    */
   function humanID (x) {
     // both events and metadata toString() return the metadata
```

Comment:
comments needs update too

Reply:
will do this is one of the next two PRs. git is having trouble checking out the `plan-9` branch due to some file permissions and it's just going to be easier to add it later. thanks for catching it.